larky-api/pom.xml

```diff
@@ -45,8 +45,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>${lombok.version}</version>
-            <scope>compile</scope>
```

Comment:
I assume you still want the `compile` scope?  Otherwise this will be pulled in as a transitive dependency

Reply:
My mistake, this should be `provided` as changed below